react-components/src/hooks/useSceneDefaultCamera.tsx

```diff
@@ -44,7 +44,14 @@ export const useSceneDefaultCamera = (
 
     return {
       fitCameraToSceneDefault: () => {
+        const initialCameraState = viewer.cameraManager.getCameraState();
+
         viewer.cameraManager.setCameraState({ position, target });
+
+        // Keep the camera target (pivot) in the same position
+        viewer.cameraManager.setCameraState({
+          target: initialCameraState.target
```

Comment:
`initialCameraState.target` value be at origin when the scene is loaded for the first time, it would set the pivot point to origin when scene loaded for the first time

Reply:
Good point, that would be an issue if the models are really far away from origin

Reply:
Checking for camera being in origin in [ea7034a](https://github.com/cognitedata/reveal/pull/4686/commits/ea7034a9e9f4623f5fac913bb95b59daaeefdfa3)